File: Completed/CSharp/1.cs
public class Solution {
    public int[] TwoSum(int[] nums, int target) {
        List<int> solution = new List<int>();
        
        for(int n = 0; n < nums.Length; n++){
            for(int m = 0; m < nums.Length; m++){
                if(n != m && nums[n] + nums[m] == target) {
                    solution.Add(n);
                    solution.Add(m);
                    return solution.ToArray();
                } 
            }
        }
        
        return solution.ToArray();
        
    }
}

File: Completed/Go/1.go
func twoSum(nums []int, target int) []int {
	m := map[int]int{}

	for i, num := range nums {
		val, ok := m[num]
		if ok && i != val {
			return []int{i, val}
		}

		comp := target - num
		m[comp] = i
	}

	return []int{-1, -1}
}